app/test_main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from unittest import mock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "has_net_connection, valid_url",
+    [
+        (True, True),
+        (6, True),
```

Comment:
why do you use numbers? use different mocks like False, True or False, False

Reply:
Okey, than I should cereate one more test with False conditions and catch an errors?